MAINTAINERS.md

```diff
@@ -6,5 +22,91 @@
 | Peter Nied | [peternied](https://github.com/peternied) | Amazon |
 | Barani Bikshandi | [bbarani](https://github.com/bbarani) | Amazon |
 | Sayali Gaikawad | [gaiksaya](https://github.com/gaiksaya) | Amazon |
+| Abhinav Gupta | [abhinavGupta16](https://github.com/abhinavGupta16) | Amazon |
+| Zelin Hao | [zelinh](https://github.com/zelinh) | Amazon |
+| Elaine | [zylzulu](https://github.com/zylzulu) | Amazon |
+
+[This document](https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md) explains what maintainers do in this repo, and how they should be doing it. If you're interested in contributing, see [CONTRIBUTING](CONTRIBUTING.md).
+
+## Zenhub Process Workflow
+We follow agile methodologies for our development and release process. We use zenhub to manage and track our stories and issues to effectively manage them over the sprint.
+
+### What is [Zenhub](https://www.zenhub.com/)?
+
+ZenHub is an agile project management and product roadmaps solution, natively integrated into GitHub. It is free to use for opensource repository and comes with a paid membership to manage private repositories. 
+We currently use zenhub only with our public and opensource repositories.
+
+### Setting up Zenhub
+
+Zenhub can be easily added as an [extension]((https://www.zenhub.com/extension)) to chrome and firefox which can be downloaded for free from the zenhub website. 
+Alternatively, we can use a the zenhub [webapp link](https://app.zenhub.com/workspaces/engineering-effectiveness-614cf4272a385f0015d2b48f/board?repos=357723952,406037663) to view the board.
+
+### Sprint Board aka Zenhub board aka Kanban board
+Once the zenhub extension is installed, zenhub board can be accessed using the Zenhub tab on github.
+
+#### Pipelines
+
+1. **New Issues -** Issues to be reviewed and estimated before being added to the Product Backlog.
+2. **Icebox -** Low priority Issues that do not need to be addressed in the near future**.**
+3. **Product Backlog -** Upcoming Issues that have been reviewed, estimated, and prioritized top-to-bottom.
+4. **Sprint Backlog -** Issues ready to be worked on in the sprint, prioritized top-to-bottom.
+5. **In Progress -** Issues currently being worked on by the team.
+6. **Review/QA -** Issues open to the team for review and testing. Code complete, pending feedback.
+7. **Done -** Issues tested and ready to be deployed to production. Verify the acceptance criteria and close the issue.
+8. **Closed -** Issues that are deployed to production and closed
+
+Description for each pipeline can also be found on the sprint board by clicking on the 3 dots next to the pipleine name.
+
+### Creating Issues
+
+Follow the steps below to create issues on Zenhub workflow -
+
+1. Create the issue for the desired repository following the required guidelines for mandatory fields on github.
```

Comment:
Lets make these instructions specific with exactly what is required for our process, sub-bullets might work if more explanation is needed, as we have optional fields that are used by contributors creating issues but for maintainers they are required.
```suggestion
1. Create the issue for the OpenSearch-build repository following with the fields `foo`, `bar`, and `baz`...
```

Reply:
I don't think we should have specifics here since this will vary from repo to repo. I changed it to `mandatory and optional fields`. Does that work?

Reply:
Where are the instructions for this repository?  This file is located in the opensearch-build repository which I would expect to be specific

Reply:
Instructions show up when someone tries to create an issue.

Reply:
Those are instructions to create issues in our repository, which are not the same as work items that we managed in our sprints - I would like there to be a guide for what GitHub/ZenHub fields we use where would that be if not in this document?